### PR TITLE
test: Add hub-auth to bdd test

### LINKED ISSUE #138: Add hub-auth to bdd test

### PR DESCRIPTION
**Pull request by fqutishat**
closes #138

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: test/bdd/mock/loginconsent/main.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	listenAddr := os.Getenv("LISTEN_ADDR")
+	if listenAddr == "" {
+		return nil, fmt.Errorf("env variable LISTEN_ADDR required")
+	}
+
+	prov := memstore.NewProvider()
+
+	err = prov.CreateStore("bdd_tests")
+	if err != nil {
+		return nil, fmt.Errorf("could not create memory store for some reason: %w", err)
```
Comment (DRK3) [inline test/bdd/mock/loginconsent/main.go:89]: The "for some reason" part sounds sort of odd/casual for an error message to print out. You can just say "could not create memory store: %w"

Comment (fqutishat) [reply test/bdd/mock/loginconsent/main.go:89]: I used this code from hub-auth

**Comment by DRK3** [comment]
I didn't think that hub-auth would be used directly with an EDV server... but I might be mistaken... @llorllale, is this how it's supposed to work?

File: test/bdd/mock/loginconsent/main.go (hunk 1 above)
Comment (fqutishat) [reply test/bdd/mock/loginconsent/main.go:89]: But I agree the msg is wrong

**Comment by fqutishat** [comment]
> I didn't think that hub-auth would be used directly with an EDV server... but I might be mistaken... @llorllale, is this how it's supposed to work?

I need hub-auth for testing zcap feature

File: test/bdd/mock/loginconsent/main.go (hunk 1 above)
Comment (fqutishat) [reply test/bdd/mock/loginconsent/main.go]: done

**Review by grace0925 — APPROVED** [review]

**Review by DRK3 — APPROVED** [review]

**Review by sandrask — APPROVED** [review]

**Comment by llorllale** [comment]
@DRK3 

> I didn't think that hub-auth would be used directly with an EDV server... but I might be mistaken... @llorllale, is this how it's supposed to work?

I'm not aware of any requirement to make hub-auth communicate with an EDV server.

**Pull request merged**